test/test_suite.cpp

```diff
@@ -1019,7 +1019,7 @@ TEST_CASE("string length", "[]" ) {
   PRINTING_CHECK("123", ==, test::sprintf_, buffer, "%.7s", "123");
   PRINTING_CHECK("", ==, test::sprintf_, buffer, "%.7s", "");
   PRINTING_CHECK("1234ab", ==, test::sprintf_, buffer, "%.4s%.2s", "123456", "abcdef");
-  PRINTING_CHECK(".2s", ==, test::sprintf_, buffer, "%.4.2s", "123456");
```

Comment:
Please remove this from the PR, it's a separate issue.

Reply:
Huh, thought I did, but will do!